doc-content/enterprise-only/openshift/rhdm71-authoring-ha-ref.adoc

```diff
@@ -0,0 +1,500 @@
+.7+| `${APPLICATION_NAME}-rhdmindex`
+|`ES_CLUSTER_NAME` | -- | `${APPFORMER_ELASTIC_CLUSTER_NAME}`
+|`ES_NODE_NAME` | Sets the ES node.name property. Defaults to HOSTNAME env value. | `${ES_NODE_NAME}`
+|`ES_TRANSPORT_HOST` | Sets the ES transport.host property. This will set the transport address of the main ES cluster node. Used for communication between nodes in the cluster. Defaults to container address. | `${ES_TRANSPORT_HOST}`
+|`ES_TRANSPORT_TCP_PORT` | -- | `${APPFORMER_ELASTIC_PORT}`
+|`ES_HTTP_PORT` | Sets the ES http.port property. This will set the http port of the main ES cluster node. Used to interact with cluster rest api. | `${ES_HTTP_PORT}`
+|`ES_HTTP_HOST` | Sets the ES http.host property. This will set the http address of the main ES cluster node. Used to interact with cluster rest api. Defaults to the container ip address | `${ES_HTTP_HOST}`
+|`ES_JAVA_OPTS` | Appends custom jvm configurations/properties to ES jvm.options configuration file. | `${ES_JAVA_OPTS}`
+.6+| `${APPLICATION_NAME}-amq`
+|`AMQ_USER` | -- | `${APPFORMER_JMS_BROKER_USER}`
+|`AMQ_PASSWORD` | -- | `${APPFORMER_JMS_BROKER_PASSWORD}`
+|`AMQ_ROLE` | User role for standard broker user. | `${AMQ_ROLE}`
+|`AMQ_NAME` | The name of the broker | `${AMQ_NAME}`
+|`AMQ_TRANSPORTS` | -- | openwire
+|`AMQ_GLOBAL_MAX_SIZE` | Maximum amount of memory which message data may consume (Default: Undefined, half of the system's memory). | `${AMQ_GLOBAL_MAX_SIZE}`
+|=======================================================================
+
+
+
+=====  Volumes
+
+|=============
+|Deployment |Name  | mountPath | Purpose | readOnly 
+
+|`${APPLICATION_NAME}-rhdmcentr` | decisioncentral-keystore-volume | `/etc/decisioncentral-secret-volume` | ssl certs | True
+|`${APPLICATION_NAME}-kieserver` | kieserver-keystore-volume | `/etc/kieserver-secret-volume` | ssl certs | True
+|`${APPLICATION_NAME}-rhdmindex` | `${APPLICATION_NAME}-rhdmindex-pvol` | `/opt/elasticsearch/data` | rhdmindex | false
+|=============
+
+
+=== External Dependencies
+
+
+==== Volume Claims
+
+A `PersistentVolume` object is a storage resource in an OpenShift cluster. Storage is provisioned by an administrator
+by creating `PersistentVolume` objects from sources such as GCE Persistent Disks, AWS Elastic Block Stores (EBS), and NFS mounts.
+Refer to the https://docs.openshift.com/enterprise/3.0/dev_guide/persistent_volumes.html#overview[Openshift documentation] for
+more information.
+
+|=============
+|Name | Access Mode
+
+|`${APPLICATION_NAME}-rhdmcentr-claim` | ReadWriteMany
+|`${APPLICATION_NAME}-rhdmindex-claim` | ReadWriteOnce
+|=============
+
+
+
+
+==== Secrets
+
+This template requires the following secrets to be installed for the application to run.
```

Comment:
Bullet these, even when only one probably. Same for all sections like it. Comes up a few times.

Reply:
Again I am very uneasy about changing autogenerated files. I will see what I can do.